Ombi.Core/HeadphonesSender.cs

```diff
@@ -88,87 +88,20 @@ public HeadphonesSender(IHeadphonesApi api, HeadphonesSettings settings, IReques
         {
             var index = await Api.GetIndex(Settings.ApiKey, Settings.FullUri);
             var artistExists = index.Any(x => x.ArtistID == request.ArtistId);
+            bool artistAdd = false;
             if (!artistExists)
             {
-                var artistAdd = Api.AddArtist(Settings.ApiKey, Settings.FullUri, request.ArtistId);
```

Comment:
@tidusjar 

Just to comment here...  This was the main defect associated much of the HP integration issues.  `artistAdd` was getting assigned a `Task<bool>` and things didn't really fall into place until that task was complete -- all of the extra polling and retrying was just unnecessary.

A convention we use at my day job is to **not** use `var` like this so frequently.  Variable types should always be explicitly obvious on the line of their declaration.  Had this code been explicit about `bool artistAdd` from the start, the code wouldn't have even compiled.

The only time I am a fan of `var` is when things are super obvious like:
```var thing = new Thing();```
or
```var something = GetObject<Something>(param1, param2);```

Just thought I'd share that as I think it helps code be more readable and prevent odd behaviors when you get unexpected types returns.

Cheers!